src/core/linter-rules/check-period-in-p.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Linter rule "check-period". Makes sure the there are no periods missing at the end of a <p>
+ *   in the ReSpec config.
+ */
+import { lang as defaultLang } from "core/l10n";
+import LinterRule from "core/LinterRule";
+
+const name = "period-in-p";
+
+const meta = {
+  en: {
+    description: "`<p>` tags should end with a period",
+    howToFix: "Please put a period at the end of this `<p>` tag",
+  },
+};
+
+// Fall back to english, if language is missing
+const lang = defaultLang in meta ? defaultLang : "en";
+
+/**
+ * Runs linter rule.
+ *
+ * @param {Object} config The ReSpec config.
+ * @param  {Document} doc The document to be checked.
+ */
+function lintingFunction(conf, doc) {
+  
+  const offendingMembers = Array.from(doc.querySelectorAll("p"))
+    // this gives us all paragraphs that don't end with a period
+    .filter(key => !key.textContent.trim().split("tests")[0].endsWith("."))
```

Comment:
split("tests")[0] ?

Was expecting: 
```js
.filter(elem => elem.textContent.trim().endsWith("."))
```

Reply:
I had initially written a patch at #1487 . Thus, when tests exist, the `textContent()` also contains the word `tests`. This I guess should not be a part of the innerText and thus, should be taken care of before analysing so as to avoid unnecessary cases.

Reply:
Should have included "!" in the code above, fixed in comment.